lib/services/api/api_service.ts

```diff
@@ -16,9 +16,13 @@ const REQUEST_PATH_REGISTER = '/auth';
 const REQUEST_PATH_LOGIN = '/auth/sign_in';
 const REQUEST_PATH_CHANGE_PW = '/auth/change_pw';
 const REQUEST_PATH_SYNC = '/items/sync';
+const REQUEST_PATH_LOGOUT = '/auth/sign_out';
```

Comment:
Isn't auth/sign_out just destroying the session? In that case, shouldn't this be a DELETE request to /session?

Reply:
The `DELETE` request to `/session` can destroy any session for the current user, except the current session.